src/common/Finisher.cc

```diff
@@ -60,25 +58,30 @@ void *Finisher::finisher_thread_entry()
       finisher_lock.Unlock();
       ldout(cct, 10) << "finisher_thread doing " << ls << dendl;
 
+      if(logger)
+        start = ceph_clock_now(cct);
+
       // Now actually process the contexts.
       for (vector<Context*>::iterator p = ls.begin();
-	   p != ls.end();
-	   ++p) {
-	if (*p) {
-	  (*p)->complete(0);
-	} else {
-	  // When an item is NULL in the finisher_queue, it means
-	  // we should instead process an item from finisher_queue_rval,
-	  // which has a parameter for complete() other than zero.
-	  // This preserves the order while saving some storage.
-	  assert(!ls_rval.empty());
-	  Context *c = ls_rval.front().first;
-	  c->complete(ls_rval.front().second);
-	  ls_rval.pop_front();
-	}
-	if (logger) {
-	  logger->dec(l_finisher_queue_len);
-          logger->tinc(l_finisher_complete_lat, ceph_clock_now(cct) - start);
+           p != ls.end();
```

Comment:
I find in this file, some places use space, and some places use tab. I also looked up other files(for example: AsyncMessenger.cc), they use "space". So I change it to keep consistent.